### PR TITLE
jsonnet: disable insecure cypher suites for prometheus-adapter

### PR DESCRIPTION
**Pull request by fpetkovski**
Running [sslscan](https://github.com/rbsec/sslscan) against the prometheus adapter secure port reports two
insecure SSL ciphers, `ECDHE-RSA-DES-CBC3-SHA` and `DES-CBC3-SHA`.

This commit removes those ciphers from the list.

Signed-off-by: fpetkovski <filip.petkovsky@gmail.com>

### CODE REVIEW

File: jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet
Comment (paulfantom) [inline jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet:148]: This should be configurable on higher level. Something like https://github.com/prometheus-operator/kube-prometheus/blob/main/jsonnet/kube-prometheus/components/kube-rbac-proxy.libsonnet#L11-L38 should work.

Comment (fpetkovski) [reply jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet:148]: Should we have a single variable to control the cipher suites for all components?

Comment (fpetkovski) [reply jsonnet/kube-prometheus/components/prometheus-adapter.libsonnet:148]: Hm nevermind, that won't work since components can be referenced directly, bypassing main

**Review by paulfantom — APPROVED** [review]
lgtm

leaving final say to @dgrisonnet as a maintainer of prometheus-adapter ;)

**Comment by dgrisonnet** [comment]
LGTM, thanks @fpetkovski :+1:

**Pull request merged**
